ballerina/tests/gsheets-composite-key-tests.bal

```diff
@@ -0,0 +1,218 @@
+        test:assertEquals(orderItem.message(), "Invalid key: {\"orderId\":\"invalid-order-id\",\"itemId\":\"item-1\"}");
+    } else {
+        test:assertFail("Error expected.");
+    }
+
+    
+}
+
+@test:Config {
+    groups: ["composite-key", "google-sheets"],
+    dependsOn: [gsheetsCompositeKeyCreateTest],
+    enable: true
+}
+function gsheetsCompositeKeyReadOneTestNegative2() returns error? {
+    // GoogleSheetsRainierClient rainierClient =  check new ();
+    OrderItem|error orderItem = rainierClient->/orderitems/[orderItem1.orderId]/["invalid-item-id"].get();
+    if orderItem is persist:NotFoundError {
+        test:assertEquals(orderItem.message(), "Invalid key: {\"orderId\":\"order-1\",\"itemId\":\"invalid-item-id\"}");
+    } else {
+        test:assertFail("Error expected.");
+    }
+
+    
+}
+
+@test:Config {
+    groups: ["composite-key", "google-sheets"],
+    dependsOn: [gsheetsCompositeKeyCreateTest, gsheetsCompositeKeyReadOneTest, gsheetsCompositeKeyReadManyTest, gsheetsCompositeKeyReadOneTest2],
+    enable: true
+}
+function gsheetsCompositeKeyUpdateTest() returns error? {
+    // GoogleSheetsRainierClient rainierClient =  check new ();
+    OrderItem orderItem = check rainierClient->/orderitems/[orderItem2.orderId]/[orderItem2.itemId].put({
+        quantity: orderItem2Updated.quantity,
+        notes: orderItem2Updated.notes
+    });
+    test:assertEquals(orderItem, orderItem2Updated);
+
+    orderItem = check rainierClient->/orderitems/[orderItem2.orderId]/[orderItem2.itemId].get();
+    test:assertEquals(orderItem, orderItem2Updated);
+
+    
+}
+
+@test:Config {
+    groups: ["composite-key", "google-sheets"],
+    dependsOn: [gsheetsCompositeKeyCreateTest, gsheetsCompositeKeyReadOneTest, gsheetsCompositeKeyReadManyTest, gsheetsCompositeKeyReadOneTest2],
+    enable: true
+}
+function gsheetsCompositeKeyUpdateTestNegative() returns error? {
+    // GoogleSheetsRainierClient rainierClient =  check new ();
+    OrderItem|error orderItem = rainierClient->/orderitems/[orderItem1.orderId]/[orderItem2.itemId].put({
+        quantity: 239,
+        notes: "updated notes"
+    });
+    if orderItem is persist:NotFoundError {
+        test:assertEquals(orderItem.message(), "Not found: [\"order-1\",\"item-2\"]");
+    } else {
+        test:assertFail("Error expected.");
+    }
+
+    
+}
+
+@test:Config {
+    groups: ["composite-key", "google-sheets"],
+    dependsOn: [gsheetsCompositeKeyUpdateTest],
+    enable: true
+}
+function gsheetsCompositeKeyDeleteTest() returns error? {
+    // GoogleSheetsRainierClient rainierClient =  check new ();
+    OrderItem orderItem = check rainierClient->/orderitems/[orderItem2.orderId]/[orderItem2.itemId].delete();
+    test:assertEquals(orderItem, orderItem2Updated);
+
+    OrderItem|error orderItemRetrieved = rainierClient->/orderitems/[orderItem2.orderId]/[orderItem2.itemId].get();
+    test:assertTrue(orderItemRetrieved is persist:NotFoundError);
+
+    
```

Comment:
```suggestion
```

Reply:
fixed in [585c1fe](https://github.com/ballerina-platform/module-ballerinax-persist.googlesheets/pull/1/commits/585c1fe07a50dd99b0f360006cc9450193cc7373)